### PR TITLE
Meetings and Questions model

### PR DESCRIPTION
**Pull request by matthewgilligan**

### CODE REVIEW

File: app/models/meeting.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Meeting < ApplicationRecord
+  belongs_to :interviewee, class_name:"User"
+  belongs_to :interviewer, class_name:"User"
+
+  validates :start_date, presence: true
```
Comment (lucien-george) [inline app/models/meeting.rb:5]: maybe also validate that the `start_date` is in the future?

File: app/models/question.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Question < ApplicationRecord
+  has_many :meetings
+
+  validates :industry, presence: true
+  validates :type, presence: true
```
Comment (lucien-george) [inline app/models/question.rb:5]: Can user enter any `industry` and `type` or do we want to restrict the values they can enter?

File: app/models/meeting.rb (hunk 1 above)
Comment (matthewgilligan) [reply app/models/meeting.rb:5]: yeah nice one

File: app/models/question.rb (hunk 2 above)
Comment (matthewgilligan) [reply app/models/question.rb:5]: maybe we could do an array - similar to the Categories in the Toys R US?

Comment (lucien-george) [reply app/models/question.rb:5]: exactly what I had in mind

**Pull request merged**